pycontrails/physics/jet.py

```diff
@@ -24,14 +24,62 @@
 # -------------------
 
 
+def acceleration(
+    true_airspeed: npt.NDArray[np.float_], dt: npt.NDArray[np.float_]
+) -> npt.NDArray[np.float_]:
+    r"""Calculate the acceleration/deceleration at each waypoint.
+
+    Parameters
+    ----------
+    true_airspeed : npt.NDArray[np.float_]
+        True airspeed, [:math:`m \ s^{-1}`]
+    dt : npt.NDArray[np.float_]
+        Time between waypoints, [:math:`s`]
+
+    Returns
+    -------
+    npt.NDArray[np.float_]
+        Acceleration/deceleration, [:math:`m \ s^{-2}`]
+    """
+    dv_dt = np.empty_like(true_airspeed)
+    dv_dt[:-1] = np.diff(true_airspeed) / dt[:-1]
+    dv_dt[-1] = 0.0
+    np.nan_to_num(dv_dt, copy=False)
+    return dv_dt
+
+
+def climb_descent_angle(
+    true_airspeed: npt.NDArray[np.float_], rocd_ms: npt.NDArray[np.float_]
```

Comment:
Are we actually using rocd in meters / s?  I found when I refactored this last time that we converted back and forth multiple times, so I actually stuck with `rocd` in `feet / min`. See e.g. `jet.reserve_fuel_requirements`.  

If you really get rocd in meters / second and don't re-convert it elsewhere, then I think its fine to have both types of units.

Ideally we'd keep all these units the same. If you go back to `ft / min`, then I would add a "See Also" for `flight.segment_rocd`:

```
    See Also
    --------
    :func:`flight.segment_rocd`
    :func:`flight.segment_true_airspeed`
```

Reply:
We do need rocd in m/s because the unit must agree with `true_airspeed`. But you're absolutely right that having several ROCDs is confusing. Because we don't use `rocd_ms` elsewhere, I'll make the conversion internal to this function.

Reply:
ae912d7